### PR TITLE
Fixed #64 Fixed #54

### LINKED ISSUE #54: User slug generator
Set user slug based on first_name and last_name.

**Example:**
```ruby
first_name: 'Volodya'
second_name: 'Sveredyuk'
slug: 'volodya-sveredyuk' # or volodya-sveredyuk-1 or -2 or -3 if duplicated
```

### LINKED ISSUE #64: Members list
Ability to see users list with first and last name

### PR DESCRIPTION
**Pull request by denys-medynskyi**

### CODE REVIEW

File: app/helpers/navigation_helper.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def chat_link
   end
 
   def members_link
-    link_tu t('members.plural'), members_path
```
Comment (sveredyuk) [inline app/helpers/navigation_helper.rb:19]: link_ту.... лол

File: spec/features/users/sign_up_spec.rb [hunk 2]
```diff
@@ -22,6 +22,27 @@
...
+        fill_in 'First name', with: first_name
+        fill_in 'Last name', with: last_name
+        fill_in 'Email', with: email
+        fill_in 'Password', with: '123456', match: :prefer_exact
+        fill_in 'Password confirmation', with: '123456', match: :prefer_exact
+
+        expect { click_button 'Sign up' }.to change(User, :count).by(1)
+
+        new_user = User.last
+        expect(new_user.slug).to eq("#{new_user.first_name.downcase}-#{new_user.last_name.downcase}-2")
```
Comment (sveredyuk) [inline spec/features/users/sign_up_spec.rb:43]: I think dot better than dash, event for SEO

'volodya.sveredyuk' is much better than 'volodya-sveredyuk'

@denysmedynskyi what do you think ?

File: spec/features/users/sign_up_spec.rb [hunk 3]
```diff
@@ -22,6 +22,27 @@
...
       expect(new_user.email).to eq(email)
     end
+
+    context 'when creating user with the same first_name and last_name' do
+      it 'creates user with correct slug' do
+        first_name = 'Denys'
+        last_name = 'Medynskyi'
+        email = Faker::Internet.email
+
+        FactoryGirl.create(:user, first_name: first_name, last_name: last_name)
```
Comment (sveredyuk) [inline spec/features/users/sign_up_spec.rb]: just `create`, rspec included FactortGirl DSL already

**Comment by sveredyuk** [comment]
@denysmedynskyi Good Job.

**Review by sveredyuk — APPROVED** [review]

File: spec/features/users/sign_up_spec.rb (hunk 2 above)
Comment (denys-medynskyi) [reply spec/features/users/sign_up_spec.rb:43]: `.` is more readable, agree. I don't know anything about SEO, so I will agree on anything)

Comment (sveredyuk) [reply spec/features/users/sign_up_spec.rb:43]: Google says dash is better that dot. Sorry, dash is ok.

File: spec/features/users/sign_up_spec.rb (hunk 3 above)
Comment (denys-medynskyi) [reply spec/features/users/sign_up_spec.rb]: Thanks. Will be fixed.

**Pull request merged**